src/components/legend/draw.js

```diff
@@ -639,12 +639,12 @@ function computeLegendDimensions(gd, groups, traces) {
         });
 
         // check if legend fits in one row
-        oneRowLegend = (fullLayout.width - (fullLayout.margin.r + fullLayout.margin.l)) > borderwidth + fullTracesWidth - traceGap;
+        oneRowLegend = (fullLayout.width - (fullLayout._size.r + fullLayout._size.l)) > borderwidth + fullTracesWidth - traceGap;
```

Comment:
`fullLayout.width - (fullLayout._size.r + fullLayout._size.l) === fullLayout._size.w`

Reply:
Nothing gets by you :) Thank you again for the quick review!